app/views/section1/not_declarant.scala.html

```diff
@@ -37,7 +37,7 @@
 
 @govukLayout(
     title = Title("notDeclarent.title"),
-    backButton = Some(BackButton(messages("site.backToPreviousQuestion"), RootController.displayPage))
+    backButton = Some(BackButton(messages("site.backToSelectionPage"), RootController.displayPage))
```

Comment:
You could use here `ChoiceController.displayPage` instead of `RootController.displayPage` in order to avoid a redirection

Reply:
The RootController was already existing that's reason we didn't change it initially . We changed it now as per the comments. Thanks @udinobi